packages/adapter-cloudflare-workers/package.json

```
@@ -38,14 +38,15 @@
 		"check": "tsc --skipLibCheck"
 	},
 	"dependencies": {
-		"@cloudflare/workers-types": "^4.20231121.0",
+		"@cloudflare/workers-types": "^4.20250129.0",
 		"@iarna/toml": "^2.2.5",
+		"crossws": "^0.3.2",
```

Bump the minimum crossws version to the one that supports throwing in upgrade https://github.com/unjs/crossws/releases/tag/v0.3.3
```suggestion
		"crossws": "^0.3.3",
```